clang/lib/Headers/avxintrin.h

```diff
@@ -2180,7 +2180,8 @@ _mm256_cvtepi32_pd(__m128i __a)
   return (__m256d)__builtin_convertvector((__v4si)__a, __v4df);
 }
 
-/// Converts a vector of [8 x i32] into a vector of [8 x float].
+/// Converts a vector of [8 x i32] into a vector of [8 x float]. Rounds inexact
+///    results according to the rounding control bits in the MXCSR register.
```

Comment:
I think we don't need to mention it. This is the default behavior if not mentioned otherwise.

Reply:
Discussed this internally and agreed that it is not necessary.